packages/api-generator/src/maps/$vuetify.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  $vuetify: {
```

Comment:
this file was renamed to `vuetify` because the `$` was causing problems with the new docs structure.